ci/deps/azure-36-minimum_versions.yaml

```diff
@@ -5,26 +5,23 @@ channels:
 dependencies:
   - beautifulsoup4=4.6.0
   - bottleneck=1.2.1
+  - cython>=0.29.13
   - jinja2=2.8
   - numexpr=2.6.2
   - numpy=1.13.3
   - openpyxl=2.4.8
   - pytables=3.4.2
   - python-dateutil=2.6.1
-  - python=3.5.3
+  - python=3.6.0
   - pytz=2017.2
   - scipy=0.19.0
   - xlrd=1.1.0
   - xlsxwriter=0.9.8
   - xlwt=1.2.0
   # universal
+  - html5lib=1.0.1
```

Comment:
is this updated in install.rst?

Reply:
there was no version in install.rst for it, should I add it?

Reply:
where is the whole section on installation: https://pandas.pydata.org/pandas-docs/stable/install.html

is this not in the 1.0 docs?

Reply:
It is, so far we have a link at the beginning of the top-level install document. We need to see when the integration of web and docs is made, if it makes sense to have it linked somewhere else, it's wip.

https://pandas.pydata.org/pandas-docs/stable/install.html#optional-dependencies

Reply:
where in the dev docs?

this is very important to not remove content like this

Reply:
Sorry, copied the wrong link: https://dev.pandas.io/docs/getting_started/install.html#optional-dependencies

Any idea about the error in the CI?
```
_______________ TestFrozenNDArray.test_tricky_container_to_bytes _______________
[gw0] linux -- Python 3.6.0 /home/vsts/miniconda3/envs/pandas-dev/bin/python

self = <pandas.tests.indexes.test_frozen.TestFrozenNDArray object at 0x7fd89d5b6d68>

    def test_tricky_container_to_bytes(self):
>       bytes(self.unicode_container)
E       TypeError: only integer scalar arrays can be converted to a scalar index

pandas/tests/indexes/test_frozen.py:79: TypeError
_______________ TestFrozenNDArray.test_string_methods_dont_fail ________________
[gw0] linux -- Python 3.6.0 /home/vsts/miniconda3/envs/pandas-dev/bin/python

self = <pandas.tests.indexes.test_frozen.TestFrozenNDArray object at 0x7fd89d037e80>

    def test_string_methods_dont_fail(self):
        repr(self.container)
        str(self.container)
>       bytes(self.container)
E       TypeError: only integer scalar arrays can be converted to a scalar index

pandas/tests/test_base.py:47: TypeError
```